lib/api/controllers/readController.js

```diff
@@ -25,32 +25,11 @@ module.exports = function ReadController (kuzzle) {
     return kuzzle.services.list.readEngine.count(requestObject);
   };
 
-  this.listCollections = function (requestObject, context) {
+  this.listCollections = function (requestObject) {
```

Comment:
As discussed with other team members, the listCollection action should only be restricted by the action rights. Filter collections can be more harmful as a user experience than a list of collections they can't read (as the unauthorized error will trigger to let them know).